### PR TITLE
Aurora and Kestrel updates

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

File: configs/base/packages.yaml [hunk 1]
```diff
@@ -1,46 +1,35 @@
 packages:
   ascent:
-    variants: ~fortran~openmp
+    require: "~fortran~openmp"
   exawind:
-    version: [master]
+    require: "@master"
   nalu-wind:
-    version: [master]
+    require: "@master ~trilinos-solvers"
```
Comment (psakievich) [inline configs/base/packages.yaml:7]: I don't think this should be a requirement. Too strict to require versions and variants of our own software unless we know it won't build on a machine.

File: configs/base/packages.yaml [hunk 2]
```diff
@@ -1,46 +1,35 @@
...
   amr-wind:
-    version: [main]
-    variants: +tiny_profile
+    require: "@main +tiny_profile"
```
Comment (psakievich) [inline configs/base/packages.yaml:9]: Same here.

File: configs/base/packages.yaml [hunk 3]
```diff
@@ -1,46 +1,35 @@
...
-    variants: ~fortran
-  yaml-cpp:
-    version: [0.6.3]
+  hypre:
+    require: "@develop ~fortran"
   trilinos:
-    version: [14.1.0.2023.02.28, develop]
-    variants: ~adios2~alloptpkgs~amesos~anasazi~aztec~boost~chaco~complex~debug~dtk~epetra~epetraext+exodus+explicit_template_instantiation~float~fortran~fortrilinos+glm+gtest+hdf5~hypre~ifpack~intrepid~intrepid2~isorropia+kokkos~mesquite+metis~minitensor~ml+mpi~mumps~nox~openmp~phalanx~piro~python~rol~rythmos~sacado+shards~shylu+stk~stratimikos~suite-sparse~superlu~superlu-dist~teko~tempus+teuchos+tpetra+uvm~x11~xsdkflags+zlib+zoltan+zoltan2 gotype=long cxxstd=17 build_type=Release
+    require:
+      - any_of: ["@=14.1.0.2023.02.28", "@develop"]
```
Comment (psakievich) [inline configs/base/packages.yaml:28]: This should be a range.

**Review by psakievich — CHANGES_REQUESTED** [review]

File: configs/base/packages.yaml (hunk 1 above)
Comment (jrood-nrel) [reply configs/base/packages.yaml:7]: The default is `@multiphase` without this.

Comment (jrood-nrel) [reply configs/base/packages.yaml:7]: Oh, or you mean keep it as a preference?

Comment (psakievich) [reply configs/base/packages.yaml:7]: Yeah or re order them. I know there is a way to set the default branch

**Review by psakievich — APPROVED** [review]

**Comment by jrood-nrel** [comment]
This works on Kestrel now, though it will segfault the Intel compiler on Kestrel until OpenFAST is fixed. Aurora needs some special configuration attention I will have in another PR.

**Pull request merged**